### PR TITLE
Add Nova Trustzone RSA keys support

### PR DESCRIPTION
**Pull request by isergieienkov**

### CODE REVIEW

**Review by madninja — COMMENTED** [review]
This looks mostly good.. After crypto-rs https://github.com/helium/helium-crypto-rs/pull/57 merges some small changes will be needed here

File: src/device/mod.rs [hunk 1]
```diff
@@ -55,4 +67,13 @@ mod test {
         let device: Device = "/tmp/keypair.bin".parse().expect("file device");
         assert!(matches!(device, Device::File(_)));
     }
+
+    #[cfg(feature = "nova-tz")]
+    #[test]
+    fn test_device_nova_tz() {
```
Comment (madninja) [inline src/device/mod.rs:73]: How is this test ever going to pass? The test automation isn't going to run on a device with nova-tz support right/

Comment (isergieienkov) [reply src/device/mod.rs:73]: It will work on any device. Just the file on /tmp needed

Comment (madninja) [reply src/device/mod.rs:73]: And where is this file? If I turn on tests in CI this will not pass right?

Comment (isergieienkov) [reply src/device/mod.rs:73]: Right. Same as test above

Comment (madninja) [reply src/device/mod.rs:73]: right so this can't be a test because we want to run tests as part of rust CI

Comment (isergieienkov) [reply src/device/mod.rs:73]: Actually I'm wrong. these test just test validity of device string parsing. So they both will work on any hardware and don't rely on file existence

Comment (madninja) [reply src/device/mod.rs:73]: ah ok.. I was just digging into why the tests were passing with that URL

**Review by madninja — APPROVED** [review]

**Pull request merged**